### DISCUSSION TITLE
Why does mpv switch subtitles have to stop playback and re-buffer every time. Switching subtitles in vlc is much smoother.

### DISCUSSION THREAD

**Discussion by Jhonfunk** [mpv-player/mpv#14664] [Q&A]
vlc's switching subtitles without having to interrupt playback in most cases. Apparently vcl will continue to download the buffer and look for available subtitles in the buffered data after receiving the switch subtitle command. If the subtitles are also stored sequentially, then there is no need to relocate and download them. Would very much like to see mpv improve this rigid operation of subtitle switching.

#14647

**Comment by 1000Fear** [comment]
Agreed, switching SUP subtitles in 4K UHD will cause noticeable stuttering

**Comment by geextahslex** [comment]
When using external audio tracks and using seek, also causes stutters
